### DISCUSSION TITLE
Compile issues?

### DISCUSSION THREAD

**Discussion by 16sth** [afpineda/OpenSourceSimWheelESP32#14] [Q&A] [answered]
Hi afpineda,
This looks awesome as I've been trying to have a wireless button box with ESP for a while now. But I can't code, so this looks to be the right fit for me.
I've tried and failed to compile for Set Up 2. I did the powershell link step but when I compile for my ESP32, it gives errors.

I hope you can help as this would be great software.

Cheers.

[error.txt](https://github.com/user-attachments/files/16942873/error.txt)

**Comment by afpineda** [comment]
Thanks for reporting, @16sth.

I have been experiencing the same issue in other projects, too.
It seems that Arduino IDE moved to a new compiler version very sensitive to missing headers pointing to the standard C++ library.
Will fix it at soon as I can.

**Reply by 16sth** [reply]
Thanks afpineda,

I thought I was going mad. I followed your instructions perfectly.
Is there an older Arduino IDE that would be better right now?

**Reply by afpineda** [reply]
> I thought I was going mad. I followed your instructions perfectly. Is there an older Arduino IDE that would be better right now?

Latest Arduino IDE is best. 
However, I like [VSCode](https://code.visualstudio.com/) with Arduino extension, which is older and deprecated, but it works OK for now.

**Comment by afpineda** [comment, marked as answer]
Hi, @16sth.

Setup2.ino compiles correctly for me, both in Arduino IDE 2.3.2 and VSCode.

It seems that you forgot to  run **MakeSymLinks.ps1** first.
Look for error messages when running that script, too.

You must run that script after downloading or updating your clone of this project (just once).
Otherwise, the compiler is unable to find the proper files.

Please, run the script and try again. 
Then, report your findings.
